site/docs/api/error-handling.md

```diff
@@ -0,0 +1,128 @@
+---
+title: Error Handling Strategies
+description: Ways of coping with unexpected error in Hazelcast Jet.
+---
+
+Jet may experience errors on multiple levels:
+
+* on the *cluster level*: nodes can be removed from or added to the
+  cluster at any given time, due to, for example, internal errors, loss
+  of networking, or even scheduled maintenance
```

Comment:
- scheduled maintenance isn't an error condition: you can do an orderly shutdown